### DISCUSSION TITLE
Sale Price functionality included in Simple Commerce?

### DISCUSSION THREAD

**Discussion by CodePranjal** [duncanmcclean/simple-commerce#1151] [Feature Requests]
Hello,

I noticed that there is an option to input the sale price in the control panel. However, I couldn't find any calculations or features that utilize the sale price. Could you please confirm if this functionality is included in Simple Commerce? We would need it for our purposes.

Thank you!

**Comment by duncanmcclean** [comment]
Sorry, can you clarify what you mean by "sale price".

Simple Commerce only has one "Price" field on products.

**Reply by CodePranjal** [reply]
when product discount added, if it has functionality which calculates using the discount price?

**Reply by duncanmcclean** [reply]
Oh, I think I see what you mean.

The coupon/disconting calculatuions happens in the [`CouponCalculator`] class, it calculates the discount based on how the coupon is configured.

**Reply by CodePranjal** [reply]
That means, when a product is on sale (discount on products) (even when a user don't use coupon) it will calculate using the discounted/sale price instead of regular price?
